trade-finance-manager-ui/server/controllers/case/amendments/leadUnderwriter.controller.js

```diff
@@ -88,16 +90,30 @@ const getAssignAmendmentLeadUnderwriter = async (req, res) => {
  */
 const postAssignAmendmentLeadUnderwriter = async (req, res) => {
   const { _id: dealId, amendmentId, facilityId } = req.params;
-  const { data: amendment, status } = await api.getAmendmentById(facilityId, amendmentId);
+  try {
+    const { data: amendment, status } = await api.getAmendmentById(facilityId, amendmentId);
 
-  if (!amendment?.amendmentId || status !== 200) {
-    return res.redirect('/not-found');
-  }
+    if (!amendment?.amendmentId || status !== 200) {
+      return res.redirect('/not-found');
+    }
+
+    const { assignedTo: assignedToValue } = req.body;
+    // assignedToValue is only the id so have to get the user's name from api call
+    const user = await api.getUser(assignedToValue);
 
-  const { assignedTo: assignedToValue } = req.body;
-  const update = { leadUnderwriterId: assignedToValue };
+    // if no user.firstName or lastName, then values set to null
+    const updateUser = {
```

Comment:
Shall we rename this variable to `leadUnderwriter` (if no conflicts)?

Reply:
done